README.md

```diff
@@ -46,17 +46,25 @@ Check the provider documentation for provider-specific files and configuration t
 Start the deployment process
 
 ```
-$ docker run -ti -v `pwd`:/root/data docker.io/osevg/openshifter create cluster01
+$ docker run -ti -v (path to your directory):/root/data docker.io/osevg/openshifter create cluster01
 ```
 
+__NOTE__: If running `docker` directly from the same folder where descriptors and SSH keys are located, you can pass in ``pwd`` as path to your directory, e.g.
+
+    $ docker run -ti -v `pwd`:/root/data ...
+
+
 ## Provider documentation
 
 ### GCE
 
 Following are instructions required to get OpenShift working on top Google Cloud Platform (GCP).
 
-First, you need to get a GCP account and create a project for running OpenShift on it.
-Creating the project can be done using the GCP user interface or via the [`gce/create-project-gce.sh`](gce/create-project-gce.sh) script.
+First install the [Google Cloud SDK](register for an account in Google Cloud ()) in your local environment. 
```

Comment:
You do not have to, unless you want to do stuff from command line, i.e. if you use only web interface, not need for this.

Reply:
True, I'll fix that right away

Reply:
Actually, we don't yet have instructions on how to do stuff with web interface and I don't really know how to do that yet, so for the time being I think this can stay as it is.

Once we have web instructions, we can adjust this accordingly.